f1cloudwatch/init.sls

```diff
@@ -13,6 +13,7 @@ logs:
 {% endfor %}
 {% endif %}
 {% if pillar.node is defined %}
+{% if pillar.node.sites is defined %}
 {% for site, name in pillar.node.sites.items() %}
```

Comment:
Seems like a lot of this would be easier to just use `pillar.get('node', {}).get('sites', {}).items()` instead of the multi-level `if` checking here. (I think you can use `pillar.get('node:sites', {})`?)

Reply:
I've tried these combo's and a few more and none of them seem to work the way we need them to.  It may be a combination of the way define pillar and the limitations of jinja and nested yaml.  Unfortunately this seems like the best way to test for the existence of node.sites.